tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2026,6 +2026,11 @@ declare namespace ts {
          */
         runWithCancellationToken<T>(token: CancellationToken, cb: (checker: TypeChecker) => T): T;
     }
+    export enum ContextFlags {
```

Comment:
I think this as well as new signature need to be internal

Reply:
Do you mean we should use something different than the `ContextFlags` when calling the code from the completion service to prevent this being type being exposed?

Reply:
I think Sheetal means to move `ContextFlags` into one of the `ts` namespaces that’s already marked `/** @internal */` and then to mark the second parameter of `TypeChecker['getContextualType']` as `/* @internal */` (or declare it as an internal overload, but the linter might call that an unnecessary overload)

Reply:
Just filed #33350. I think it would be ok to [ignore the line from the linter](https://palantir.github.io/tslint/usage/rule-flags/). @sheetalkamat?

Reply:
Yes in past we have marked the overload internal and disabled the rule for that line.

Reply:
Done. [Had to adjust the tests](https://github.com/microsoft/TypeScript/pull/32100/commits/96a782ee4c426e58d0a2ad9f00f85e86f78c010c) because of https://github.com/microsoft/TypeScript/pull/32266

Anything else you need from my side to get this merged?

Reply:
Cool, this doesn’t even touch the public API baselines now. Would be good to get an approval from another person on the team but I think this is good.